README.md

```diff
@@ -198,7 +200,10 @@ Store user credentials securely in the KeyChain.
 credentialsManager.store(credentials: credentials)
 ```
 
-#### Retrieve stored credentials and auto-renew if expired
+#### Retrieve stored credentials 
+
+Credentials will automatically be renewed (if expired) using the refresh token. The scope `offline_access` is required
+to ensure the refreshtToken is returned.
```

Comment:
remove the new line after "required". Also typo. I would use "refresh token" instead.